package.json

```
@@ -99,11 +98,11 @@
   },
   "overrides": {
     "@vue/cli-service": {
-      "mini-css-extract-plugin": "^1.6.2",
-      "@achrinza/node-ipc": "^10.1.10"
-    },
+    "mini-css-extract-plugin": "^1.6.2",
+    "@achrinza/node-ipc": "^10.1.10"
+  },
     "pretty-checkbox-vue": {
-      "vue": "^3.0.0"
+    "vue": "^3.0.0"
```

```suggestion
      "vue": "^3.0.0"
```
Looks like the indentation is borked here